src/bridge-deployer/src/cli.rs

```diff
@@ -99,9 +118,13 @@ impl Cli {
                 evm_network,
                 private_key,
                 deploy_bft,
+                &mut canister_ids,
             )
             .await?;
 
+        // write canister ids file
+        canister_ids.write()?;
```

Comment:
This is not the expected place to write the canister_ids. This operation has to be done only in the specific flow where and when the canister_ids are modified